### PR TITLE
Lukas jaronis

### PR DESCRIPTION
**Pull request by lukasjaronis**

### CODE REVIEW

File: components/Header/index.js [hunk 1]
```diff
@@ -9,4 +9,38 @@
 //  </div >
 // And add it to the DOM in the .header-container component
 
-function Header() {}
+function Header(title, date, temp) {
+
+const create = e => document.createElement(e);
```
Comment (harjothkhara) [inline components/Header/index.js:14]: nice!

File: components/Header/index.js [hunk 2]
```diff
@@ -9,4 +9,38 @@
...
+
+ // setting up new elements
```
Comment (harjothkhara) [inline components/Header/index.js:16]: great use of comments for organization

File: components/Header/index.js [hunk 3]
```diff
@@ -9,4 +9,38 @@
...
+headerDate.textContent = `Date: ${date}`;
+headerTitle.textContent = title;
+headerTemp.textContent = temp;
+
+
+return newHeader
+}
+
+const headContainer = document.querySelector('.header-container');
+headContainer.appendChild(Header('Lambda Times', 'Friday Nov 1st', '98°'));
```
Comment (harjothkhara) [inline components/Header/index.js:46]: interesting - abstracted away the use of dom properties such as innertext inside of component

File: components/Header/index.js [hunk 4]
```diff
@@ -9,4 +9,38 @@
 //  </div >
 // And add it to the DOM in the .header-container component
 
-function Header() {}
+function Header(title, date, temp) {
+
```
Comment (harjothkhara) [inline components/Header/index.js:13]: what built in JS object could you have used for the date?

**Review by harjothkhara — APPROVED** [review]

**Comment by harjothkhara** [comment]
great work Lukas! see your student dashboard for comments.

File: components/Header/index.js (hunk 4 above)
Comment (lukasjaronis) [reply components/Header/index.js:13]: Date(), which I completely forgot about haha. Good one.

**Pull request merged**